<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctor_call</class>
 <widget class="QDialog" name="doctor_call">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>931</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="name_cure">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>440</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="effects_cure">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>540</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Добавление нового вызова&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>430</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Назначенные лекарства:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_call">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>590</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить вызов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="description_cure">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>490</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>661</width>
     <height>861</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_7">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>530</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Побочные эффекты:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>880</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад к лекарствам</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_6">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>490</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Описание:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_call">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>590</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить выбранную запись</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="doctor_name">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>140</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="date">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>190</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_8">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>80</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ФИО пациента:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_9">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>130</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ФИО врача:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_10">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>180</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Дата вызова:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_name">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>90</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="symptoms">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>290</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="prescription">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>390</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_11">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>230</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Адрес вызова:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_12">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>280</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Симптомы:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_13">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>380</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Предписания:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="address">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>240</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_14">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>670</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Количество пациентов, заболевших следующей болезнью:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_15">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>760</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;За следующую дату вызовов:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_patient">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>680</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enter_patient">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>680</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввод</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enter_call">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>770</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввод</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_call">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>770</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ГГГГ-ММ-ДД</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_16">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>710</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Результат поиска:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_17">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>810</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Результат поиска:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="diagnosis">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>340</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_err_18">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>330</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Диагноз:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_text1">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>720</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_text1_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>820</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>1191</width>
     <height>931</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>name_cure</zorder>
  <zorder>effects_cure</zorder>
  <zorder>label_err_4</zorder>
  <zorder>label_err_5</zorder>
  <zorder>add_call</zorder>
  <zorder>description_cure</zorder>
  <zorder>tableView3</zorder>
  <zorder>label_err_7</zorder>
  <zorder>back</zorder>
  <zorder>label_err_6</zorder>
  <zorder>delete_call</zorder>
  <zorder>doctor_name</zorder>
  <zorder>date</zorder>
  <zorder>label_err_8</zorder>
  <zorder>label_err_9</zorder>
  <zorder>label_err_10</zorder>
  <zorder>patient_name</zorder>
  <zorder>symptoms</zorder>
  <zorder>prescription</zorder>
  <zorder>label_err_11</zorder>
  <zorder>label_err_12</zorder>
  <zorder>label_err_13</zorder>
  <zorder>address</zorder>
  <zorder>label_err_14</zorder>
  <zorder>label_err_15</zorder>
  <zorder>search_patient</zorder>
  <zorder>enter_patient</zorder>
  <zorder>enter_call</zorder>
  <zorder>search_call</zorder>
  <zorder>label_err_16</zorder>
  <zorder>label_err_17</zorder>
  <zorder>diagnosis</zorder>
  <zorder>label_err_18</zorder>
  <zorder>label_text1</zorder>
  <zorder>label_text1_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
